### PR TITLE
Feature/analyze after build

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: its/target-platform-e43/target-platform-its-e43.target [hunk 1]
```diff
@@ -17,7 +17,7 @@
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.python.pydev.feature.feature.group" version="0.0.0"/>
-<repository location="http://www.pydev.org/update_sites/5.2.0"/>
+<repository location="https://dl.bintray.com/fabioz/pydev/5.2.0"/>
```
Comment (janos-ss) [inline its/target-platform-e43/target-platform-its-e43.target]: Why is this better?

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintPostBuildListener.java [hunk 2]
```diff
@@ -48,11 +48,11 @@
 import org.sonarlint.eclipse.ui.internal.markers.ShowIssueFlowsMarkerResolver;
 import org.sonarlint.eclipse.ui.internal.util.PlatformUtils;
 
-public class SonarLintChangeListener implements IResourceChangeListener {
+public class SonarLintPostBuildListener implements IResourceChangeListener {
 
   @Override
   public void resourceChanged(IResourceChangeEvent event) {
-    if (event.getType() == IResourceChangeEvent.POST_CHANGE) {
+    if (event.getType() == IResourceChangeEvent.POST_BUILD) {
```
Comment (janos-ss) [inline org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintPostBuildListener.java]: Will this work for languages that are not built, such as JavaScript?

File: its/target-platform-e43/target-platform-its-e43.target (hunk 1 above)
Comment (henryju) [reply its/target-platform-e43/target-platform-its-e43.target]: It was failing on cix and I don't understand why. Seems related to the redirect + composite update site.

File: org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintPostBuildListener.java (hunk 2 above)
Comment (henryju) [reply org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintPostBuildListener.java]: I did a test with JavaScript, and QA is passing. Also, documentation says the POST_BUILD event should be triggered at the time build would have been done, independently of the fact a build(er) actually run.

**Pull request merged**